src/utils/webrtc/models/CallParticipantModel.js

```diff
@@ -170,11 +170,11 @@ CallParticipantModel.prototype = {
 	},
 
 	_handleNick: function(data) {
-		if (!this.get('peer') || this.get('peer').id !== data.id) {
+		// The nick could be changed even if there is no Peer object.
+		if (this.get('peerId') !== data.id) {
 			return
 		}
 
-		this.set('userId', data.userid || null)
```

Comment:
obsolete ?

Reply:
Yes, see the commit description for details: https://github.com/nextcloud/spreed/commit/e00f6627754b298edaffa135d9a20b3f8ff35e56